.nvmrc

```diff
@@ -0,0 +1 @@
+v20.10.0
```

Comment:
Do we want to be so specific? Why this particular version and not 21?

Reply:
I used this particular version as it's consistent with Runner and Validator

Reply:
I guess we can keep it as 21 if the other two use is.